Cross building for Windows with MinGW is broken

Cross building binaries from actual sources for Windows using the MinGW compiler fails with the following error: 
```
[  161s] cd /home/abuild/rpmbuild/BUILD/vsgExamples-1.1.8+20250626+git.6a5c78e/build/tests/vsgunicode && /usr/bin/i686-w64-mingw32-g++ -DVSG_SHARED_LIBRARY  -O2 -g -pipe -Wall -fexceptions --param=ssp-buffer-size=4 -mms-bitfields -O2 -g -DNDEBUG -std=gnu++17 -Wall -Wparentheses -Wno-long-long -Wno-import -Wreturn-type -Wmissing-braces -Wunknown-pragmas -Wmaybe-uninitialized -Wshadow -Wunused -Wno-misleading-indentation -Wextra -MD -MT tests/vsgunicode/CMakeFiles/vsgunicode.dir/vsgunicode.cpp.obj -MF CMakeFiles/vsgunicode.dir/vsgunicode.cpp.obj.d -o CMakeFiles/vsgunicode.dir/vsgunicode.cpp.obj -c /home/abuild/rpmbuild/BUILD/vsgExamples-1.1.8+20250626+git.6a5c78e/tests/vsgunicode/vsgunicode.cpp
[  161s] /home/abuild/rpmbuild/BUILD/vsgExamples-1.1.8+20250626+git.6a5c78e/tests/vsgunicode/vsgunicode.cpp:8:10: fatal error: Windows.h: No such file or directory
[  161s]     8 | #include <Windows.h>
[  161s]       |          ^~~~~~~~~~~
[  161s] compilation terminated.
```
The reason for this is that with this configuration all include headers for Windows must be written in lower case.

--- a/tests/vsgunicode/vsgunicode.cpp
+++ b/tests/vsgunicode/vsgunicode.cpp
@@ -5,7 +5,7 @@
 
 #ifdef _WIN32
 #define WIN32_LEAN_AND_MEAN
-#include <Windows.h>
+#include <windows.h>
 #endif
 
 template<class C1, class C2>

